packages/react-instantsearch-hooks/README.md

```diff
@@ -468,6 +468,133 @@ function Hits(props) {
 }
 ```
 
+
+### `useMenu`
+
+> `(props: UseMenuProps) => MenuRenderState`
+
+Hook to use a [menu](https://www.algolia.com/doc/api-reference/widgets/menu/js/).
+
+**Types**
+
+<details>
+<summary><code>UseMenuProps</code></summary>
+
+```ts
+type UseMenuProps = {
+  /**
+   * The name of the attribute in the records.
```

Comment:
```suggestion
   * The attribute name in the Algolia records.
```

Reply:
this is copied from InstantSearch.js, so should be changed there instead

Reply:
Ah, too bad.